styles/themes/teammember/normalize.scss

```diff
@@ -0,0 +1,186 @@
+/* http://meyerweb.com/eric/tools/css/reset/  v2.0b1 | 201101  */
```

Comment:
Did this get removed this from teammember-client? It looks like it's still being referenced in site.scss.